packages/shared/providers.yaml

```diff
@@ -1523,6 +1523,29 @@ discourse:
             title: API Key
             description: The API key for your Discourse account
 
+dixa:
+    display_name: Dixa
+    categories:
+        - support
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://${connectionConfig.apiType}.dixa.io
+        headers:
+            Authorization: ${apiKey}
```

Comment:
Can be tested maybe

Reply:
Should we enforce verification on untested integrations on our end for both [Dixa](https://github.com/NangoHQ/nango/pull/2899) and [Chattermill](https://github.com/NangoHQ/nango/pull/2900). Could this be a case of shooting ourselves in the foot 🤔.

Reply:
I mean when you publish an integration I have no way to know if it's tested or not, if you don't feel confident (that's okay) you should write it in the PR desc ☺️

Reply:
I am sorry for the miscommunication here🤦‍♂️. Thank you for the clarification 🙏.